app/renderer/components/navbar.js

```diff
@@ -95,7 +95,7 @@ module.exports = {
                     this.deleteSuite();
                     break;
                 default:
-                    this.AppStatus.events.emit(selection);
+                    AppStatus.events.emit(selection);
```

Comment:
@angrykoala - this fixes a bug with `Run Suite`

Reply:
oh, thanks, what was the bug?